src/contract_client/client.ts

```diff
@@ -45,6 +47,49 @@ export class ContractClient {
     });
   }
 
+  /**
+   * Generate a ContractClient instance from the ContractClientOptions and the wasm hash 
+   */
+  static async fromWasmHash(wasmHash: Buffer, options: ContractClientOptions): Promise<ContractClient> {
```

Comment:
Why take `wasmHash` a Buffer? Most of my experience working with wasm hashes comes from using the CLI, where I'm used to copy-pasting a string. I see even in the test here, you end up with a hex string that you then need to encode as a Buffer. Can we skip that, and pass a string here?

Reply:
Sure, we can pass a string, perhaps with an encoding option that defaults to hex?